04-a-catalogue-of-pedagogical-patterns.md

```diff
@@ -169,7 +169,7 @@ be adjusted by giving tips in response to formative assessments. Working in
 small groups can help mitigate the risks of students having trouble getting 
 started or pursuing tangents.
 
-## Twiddle, tweak, and frob
+## Tweak, twiddle, and frob
```

Comment:
I think the original intent was "twiddle" being a state of playing, "tweak" is a small change, and "frob" is a large change, so I believe the original version was the intended order. @AllenDowney: can you confirm?